app/models.py

```diff
@@ -3,14 +3,18 @@
 import hashlib
 import re
 from pathlib import Path
-
 from typing import Optional
+
 from colorfield.fields import ColorField
 from django.db import models
+from django.forms import model_to_dict
+from django.shortcuts import get_object_or_404
 from django.urls import reverse
 from django.utils.safestring import mark_safe
 from django.utils.text import slugify
 
+from scripts.hdf5.rds2hdf5 import read_from_hdf
```

Comment:
This function needs to be in a file somewhere else, it's part of the Django app. Maybe in file `app/utils/hdf5.py`? Also, the function could have a shorter name, like `read_hdf5`, similar to `pandas` convention.

```suggestion
from .utils.hdf5 import read_hdf5
```